# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-system-libpng.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-143.0.7499.40/chrome/test/BUILD.gn.vanilla	2025-12-05 18:23:48.484518472 +0100
+++ chromium-143.0.7499.40/chrome/test/BUILD.gn	2025-12-05 18:25:37.323502011 +0100
@@ -11567,7 +11567,7 @@
       "//testing/gtest",
       "//third_party/hunspell",
       "//third_party/icu",
-      "//third_party/libpng:libpng_for_testonly",
+      "//third_party/libpng",
       "//ui/base:base_interactive_ui_tests",
       "//ui/base:ozone_buildflags",
       "//ui/base:test_support",
@@ -12527,7 +12527,7 @@
       "//testing/gtest",
       "//third_party/hunspell",
       "//third_party/icu",
-      "//third_party/libpng:libpng_for_testonly",
+      "//third_party/libpng",
       "//ui/base:test_support",
       "//ui/resources:ui_test_pak",
       "//ui/views",
--- chromium-143.0.7499.40/tools/imagediff/BUILD.gn.vanilla	2025-12-05 18:25:47.969836059 +0100
+++ chromium-143.0.7499.40/tools/imagediff/BUILD.gn	2025-12-05 18:26:54.169929744 +0100
@@ -48,7 +48,7 @@
     deps = [
       "//base",
       "//build/win:default_exe_manifest",
-      "//third_party/libpng:libpng_for_testonly",
+      "//third_party/libpng",
       "//third_party/zlib",
     ]
   }
--- chromium-143.0.7499.40/ui/gfx/BUILD.gn.vanilla	2025-12-05 18:27:08.839236699 +0100
+++ chromium-143.0.7499.40/ui/gfx/BUILD.gn	2025-12-05 18:27:34.297750346 +0100
@@ -902,7 +902,7 @@
     "//skia:skcms",
     "//testing/gtest",
     "//third_party/icu:icuuc",
-    "//third_party/libpng:libpng_for_testonly",
+    "//third_party/libpng",
     "//third_party/zlib",
     "//ui/base",
     "//ui/gfx/animation",
